testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/InstrumentationSpecification.groovy

```diff
@@ -36,6 +37,13 @@ abstract class InstrumentationSpecification extends Specification {
     testRunner().clearAllExportedData()
   }
 
+  def cleanup() {
```

Comment:
Hmm, perhaps we should consider adding an `afterTest()` method to `InstrumentationTestRunner`and calling it here and in `InstrumentationExtension`?

Reply:
Thanks for pointing out InstrumentationExtension - adding afterTest seemed a bit weird since it's the same code for agent and library. For now I just copied the code since it isn't so much and I think we can revisit later.